products/batch_exports/backend/temporal/metrics.py

```diff
@@ -249,54 +251,17 @@ def reset(self):
 
 def get_metric_meter(additional_attributes: Attributes | None = None) -> MetricMeter:
     """Return a meter depending on in which context we are."""
-    attributes = get_attributes(additional_attributes)
-
     if activity.in_activity():
         meter = activity.metric_meter()
     elif workflow.in_workflow():
         meter = workflow.metric_meter()
     else:
         raise RuntimeError("Not within workflow or activity context")
 
-    meter = meter.with_additional_attributes(attributes)
-
-    return meter
-
-
-def get_attributes(additional_attributes: Attributes | None = None) -> Attributes:
```

Comment:
Getting rid of this because temporal sets `activity_type` `workflow_type` and `namespace` on all metrics made from their meters.

Reply:
From here: https://github.com/temporalio/sdk-python#metrics